frontend/src/app/shared/components/sidemenu/sidemenu.component.html

```diff
@@ -73,7 +73,7 @@
   <span
     class="op-sidemenu--item-title"
   >{{ item.title }}
-    <span class="op-sidemenu--enterprise-badge spot-icon spot-icon_enterprise-badge" *ngIf="noEEToken && item.isEnterprise"></span>
```

Comment:
I think we can delete this icon svg now, right?

Reply:
Yes, I think so if we don't need to use it in other places from now on.